README.md

```diff
@@ -59,6 +59,88 @@ and health-checking mechanisms.
   sudo yum -y install consul-1.9.1
   ```
 
+* Add yum repos:
+ 
+  * Cortx_iso.repo
+  ```sh
+  vi /etc/yum.repos.d/cortx_iso.repo
+  ```
+  * Paste:
+  ```sh
+  [cortx_iso]
+  baseurl=file:///var/artifacts/0//cortx_iso
+  gpgcheck=0
+  name=Repository cortx_iso
+  enabled=1
+  ```
+  * C7.8.2003 repo
+  ```sh
+  vi /etc/yum.repos.d/C7.8.2003.repo
+  ```
+  * Paste:
+  ```sh
+  # C7.8.2003
+  [C7.8.2003-base]
+  name=CentOS-7.8.2003 - Base
+  baseurl=http://linuxsoft.cern.ch/centos-vault/7.8.2003/os/$basearch/
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+  enabled=1
+
+  [C7.8.2003-updates]
+  name=CentOS-7.8.2003 - Updates
+  baseurl=http://linuxsoft.cern.ch/centos-vault/7.8.2003/updates/$basearch/
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+  enabled=0
+
+  [C7.8.2003-extras]
+  name=CentOS-7.8.2003 - Extras
+  baseurl=http://linuxsoft.cern.ch/centos-vault/7.8.2003/extras/$basearch/
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+  enabled=1
+
+  [C7.8.2003-centosplus]
+  name=CentOS-7.8.2003 - CentOSPlus
+  baseurl=http://linuxsoft.cern.ch/centos-vault/7.8.2003/centosplus/$basearch/
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+  enabled=1
+
+  [C7.8.2003-fasttrack]
+  name=CentOS-7.8.2003 - Fasttrack
+  baseurl=http://linuxsoft.cern.ch/centos-vault/7.8.2003/fasttrack/$basearch/
+  gpgcheck=1
+  gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
+  enabled=1
+  ```
+* Make sure lustre that "lustre-whamcloud-2.12.5" is not listed:
+  * List repos:
+  ```sh
+  yum list lustre-client-devel kmod-lustre-client 
```

Comment:
This needs to go in cortx-motr documentation and hare must add a link to motr installation document instead.

Reply:
Oh ok this makes more sense. I will do that instead. I realized the other repos are not required this was an error with VM and are not required for building HARE.

Reply:
@hessio Does it mean that we can close this PR?

Reply:
Yes, let's do that!